### PR TITLE
[BTAT-8319] - scala and sbt update

### PR DESCRIPTION
**Pull request by sraspin**

### CODE REVIEW

**Comment by sraspin** [comment]
https://github.com/hmrc/service-manager-config/pull/2603
https://github.com/hmrc/build-jobs/pull/6540

File: build.sbt [hunk 1]
```diff
@@ -115,7 +116,7 @@ lazy val microservice = Project(appName, file("."))
   .settings(defaultSettings(): _*)
   .settings(
     Keys.fork in Test := true,
-    scalaVersion := "2.11.11",
+    scalaVersion := "2.12.11",
```
Comment (JoNB7777) [inline build.sbt:119]: There's a newer version available (2.12.12)

File: build.sbt [hunk 2]
```diff
@@ -72,7 +75,7 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
 val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-26" % bootstrapPlayVersion,
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % bootstrapFrontendVersion,
   "uk.gov.hmrc" %% "govuk-template" % govTemplateVersion,
   "uk.gov.hmrc" %% "play-language" % "4.2.0-play-26",
```
Comment (JoNB7777) [inline build.sbt:80]: Is there any reason for not using the playLanguageVersion variable defined above (and in particular for using an older version here)?

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=0.13.18
+sbt.version=1.3.4
```
Comment (JoNB7777) [inline project/build.properties:1]: There's a newer version available (1.3.13)

**Review by JoNB7777 — CHANGES_REQUESTED** [review]

**Review by JoNB7777 — APPROVED** [review]

**Pull request merged**